backend/src/requests/user/requests-user.service.ts

```diff
@@ -1,19 +1,93 @@
 import {
+  HttpException,
+  HttpStatus,
   Injectable,
   InternalServerErrorException,
+  NotAcceptableException,
+  NotFoundException,
   UnauthorizedException,
 } from '@nestjs/common';
 import { LoginAttempt } from '@prisma/client';
+import { ExpertService } from '../../database/expert/expert.service';
 import { LoginAttemptService } from '../../database/login-attempt/login-attempt.service';
 import { UserService } from '../../database/user/user.service';
+import { UpdateUser } from './dto/update-user.dto';
+
+enum Registration { // TODO extract
+  registered = 'registered',
+  proUser = 'proUser',
+}
 
 @Injectable()
 export class RequestsUserService {
   constructor(
     private readonly userLoginAttemptService: LoginAttemptService,
     private readonly userService: UserService,
+    private readonly expertService: ExpertService,
   ) {}
 
+  /**
+   * Get information on a user identified by its id
+   * @returns user if found, object after the defined schema in OpenAPI
+   * @throws NotFoundException if no user found with id
+   * */
+  async getProfileWrapper(userId: string) {
```

Comment:
Maybe add return signature, what is returned in this method.
You could return the object as described in one interface